proto/executor.proto

```diff
@@ -147,6 +150,27 @@ message Join {
 	repeated Expr other_conditions = 11;
 	repeated Expr other_eq_conditions_from_in = 12;
 	optional bool is_null_aware_semi_join = 13;
+	repeated RuntimeFilter runtime_filter_list = 14; // only used by TiFlash
+}
+
+message RuntimeFilter {
+	optional int32 id = 1 [(gogoproto.nullable) = false];
+	optional Expr source_expr = 2;
+	optional Expr target_expr = 3;
+	optional string source_executor_id = 4 [(gogoproto.nullable) = false];
+	optional string target_executor_id = 5 [(gogoproto.nullable) = false];
```

Comment:
Can you give an example to explain these fields?

Reply:
I will explain it in planner code